src/java.xml/share/classes/com/sun/xml/internal/stream/writers/XMLDOMWriterImpl.java

```diff
@@ -235,7 +235,7 @@ public void writeAttribute(String namespaceURI,String localName,String value)thr
     }
 
     /**
-     * Creates a DOM Atrribute @see org.w3c.dom.Node and associates it with the current DOM element @see org.w3c.dom.Node.
+     * Creates a DOM Attribute @see org.w3c.dom.Node and associates it with the current DOM element @see org.w3c.dom.Node.
```

Comment:
Not that it matters in this PR, but I think I should mention that `@see` tags do not work like that. (No action needed from you.)

Reply:
Yeah. There's a lot of crappy code hanging around. :( Feel free to open a bug on checking incorrect usage of `@see` tags in the JDK. I'm sure you'll find more cases than those.